index.d.ts

```diff
@@ -91,6 +91,7 @@ declare module 'focus-trap' {
     deactivate(deactivateOptions?: DeactivateOptions): void;
     pause(): void;
     unpause(): void;
+    updateContainerElements(containerElements: HTMLElement | string | Array<HTMLElement | string>): void;
```

Comment:
Since the method returns the trap, return type should be `FocusTrap` instead of `void`. But then, none of the other existing API methods return the trap. I would say it's nice to enable chaining on these methods by returning the trap, including this new method, but I'm thinking it might actually be better _not_ to return the trap to keep things consistent, unless that's critical for something I'm not thinking of. All methods could be updated to return the trap in a separate PR. WDYT?

Reply:
They actually all return the trap.  They all say that in the README.md.  So the inaccurate thing here is the TS return type.

I put void cause that's what the others are doing.  But, they all should return a trap.

I don't know TS that well, so I didn't update the others.  But, now I see you already have a FocusTrap TS type, so I can actually easily change this and the others.  Let me do that real quick.

Reply:
√ Updated

Reply:
Thanks! Ah, you're right about `activate()` and `deactivate()`, but I don't see that happening for [pause and unpause](https://github.com/focus-trap/focus-trap/blob/master/index.js#L122-L133) as it is.

Do you mind adding that? Note there are 2 return paths in each method (the way they're currently implemented).

Then everything will be synced-up. 😄

Reply:
Done.  Thanks for checking.